install.sh

```diff
@@ -393,6 +393,32 @@ set_current_context_k8s() {
   fi
 }
 
+scale_down() {
```

Comment:
This is not the easiest read, would be useful to have some comments describing at least the flow.